trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable_obj.correct_name[0..9] if @nameable_obj.correct_name.length > 10
+  end
```

Comment:
- [ ] [OPTIONAL] Kindly update this method to return the same string if its length is 9 characters or less since now if the string length is less than 10 characters the function will return nil.